### PR TITLE
Added Bigtable Read DisplayData

### PR DESCRIPTION
**Pull request by ianzhou1**
Be sure to do all of the following to help us incorporate your contribution
quickly and easily:

 - [ ] Make sure the PR title is formatted like:
   `[BEAM-<Jira issue #>] Description of pull request`
 - [ ] Make sure tests pass via `mvn clean verify`. (Even better, enable
       Travis-CI on your fork and ensure the whole test matrix passes).
 - [ ] Replace `<Jira issue #>` in the title with the actual Jira issue
       number, if there is one.
 - [ ] If this contribution is large, please file an Apache
       [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.txt).

---

### CODE REVIEW

**Comment by dhalperi** [comment]
R: @swegner

File: sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/bigtable/BigtableIO.java [hunk 1]
```diff
@@ -811,6 +811,19 @@ public void validate() {
     }
 
     @Override
+    public void populateDisplayData(DisplayData.Builder builder) {
```
Comment (swegner) [inline sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/bigtable/BigtableIO.java]: Can you add a test that verifies this actually makes it through to the primitive transform? Check out [`BigQueryIOTest.testTableSourcePrimitiveDisplayData()`](https://github.com/apache/incubator-beam/blob/master/sdks/java/io/google-cloud-platform/src/test/java/org/apache/beam/sdk/io/gcp/bigquery/BigQueryIOTest.java#L648) as an example.

**Comment by ianzhou1** [comment]
R: @dhalperi @swegner

**Comment by dhalperi** [comment]
LGTM

**Comment by dhalperi** [comment]
Please backport this improvement to DataflowJavaSDK as appropriate.

**Pull request closed without merging**